### ISSUE TITLE
How to handle translations - technically possible but are we ready for maintaining translations?

### ISSUE THREAD

**Issue by sypets** [TYPO3-Documentation/TYPO3CMS-Guide-HowToDocument#161]
I am just adding this issue for easy reference ... - can be put on hold for now ...

See [Slack](https://typo3.slack.com/archives/C028JEPJL/p1595930221286000):

_My suggestion (2020-07-28)_

> There are occasionally requests for translating a manual and people offering to do this. I hate to turn them away, but I think we should discourage this if:
> * the manual in general is not in good shape: I think it should get updated and be in a good state first, before we start translating
> * there is only 1 person offering to do it - I think we should request a group of minimum 2 people maintaining it for at least a year (with one being official maintainer for the language) and commit to this

> I realize these are obstacles but we have been reluctant to create translations so I think this is in sync with previous policy but opens a small window of opportunity for someone who is really determined (currently there is a request for Russian in "Tutorial f. Editors")
See also info in "Writing documentation" https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/HowToAddTranslation/Index.html

Anyhow, focus is getting English manuals updated for now!

**Comment by sypets** [comment]
currently not relevant.
